### ISSUE TITLE
Improve bug report modal

### ISSUE THREAD

**Issue by EpokK** [ProtonMail/protonmail-settings#26]
Here are a few suggestions for the report bug button:

1) [x] Force the users to enter the username (must fill this field). Also, point to them that this is a ProtonMail username. Maybe we can write it “ProtonMail username”.

2) [x] They must select the problem from the support for which they are contacting us.
In other words, provide categories with a drop-down menu.

For example, force them to choose between:
- Login Problem
- Sign up problem
- Bridge problem
- Import/Export problem
- Custom Domains problem
- Payments problem
- VPN problem (If they select VPN, to send the inquiry directly to protonvpn.zendesk.com instead of protonmai.zendesk.com)
- Feature request
- Other

3) [x] Check if they are providing a valid contact email address (For example, they are writing `violeta` they are not writing the whole address)

4) [ ] While they are typing recognize the words and provide suggestions from the KnowledgeBase.

In other words, if they start typing “I have a login problem, show them a text in the text box “You may want to look at our most common login problems article” and include a link from it.
I know that this may take a lot of work but it will be great if it can be implemented. The suggestion can be also shown when they are selecting options from the drop-down menu.
